boskos/Makefile

```diff
@@ -73,10 +74,16 @@ metrics-deployment:
 service:
 	kubectl apply -f service.yaml
 
-update-config: get-cluster-credentials
+activate-prow:
+	if [ -a $(SERVICE_ACCOUNT) ] ; \
```

Comment:
I suggest avoiding shell in Makefiles, although this is probably simple enough. In any case, use two brackets and quote your variable: `if [[ -a $(SERVICE_ACCOUNT)" ]];`.

Reply:
(feel it's better than add another shell script)